test/Renci.SshNet.IntegrationTests/CompressionTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Renci.SshNet.Compression;
+
+namespace Renci.SshNet.IntegrationTests
+{
+    [TestClass]
+    public class CompressionTests : IntegrationTestBase
+    {
+        private IConnectionInfoFactory _connectionInfoFactory;
+
+        [TestInitialize]
+        public void SetUp()
+        {
+            _connectionInfoFactory = new LinuxVMConnectionFactory(SshServerHostName, SshServerPort);
+        }
+
+        [TestMethod]
+        public void None()
+        {
+            DoTest(new KeyValuePair<string, Func<Compressor>>("none", null));
+        }
+
+#if NET6_0_OR_GREATER
+        [TestMethod]
+        public void ZlibOpenSsh()
```

Comment:
Because the compression initialises after user authentication, are we actually testing any compressed packets? Should these tests upload/download a file, like e.g. SftpClientTest.Test_Sftp_Upload_And_Download_1MB_File?